src/System.Linq.Parallel/tests/Helpers/Functions.cs

```diff
@@ -66,6 +66,15 @@ public static void AssertAggregateAlternateCanceled(Action<CancellationToken, Ac
             Assert.All(ae.InnerExceptions, e => Assert.IsType<OperationCanceledException>(e));
         }
 
+        public static void AssertAggregateNotCanceled(Action<CancellationToken, Action> query)
+        {
+            CancellationToken token = new CancellationTokenSource().Token;
+            Action canceler = () => { throw new OperationCanceledException(token); };
```

Comment:
The point of this set of tests was to make sure it wouldn't loft any OCE it saw, but actually check that the token matched and was canceled.  So I need the token to pass to `WithCancellation` so that the token matches (but obviously isn't canceled).